### PR TITLE
feat: get case control population sizes in modal

### PR DESCRIPTION
**Pull request by pieterlukasse**
Jira Ticket: [VADC-371](https://ctds-planx.atlassian.net/browse/VADC-371)

### New Features
- show Case/Control populations sizes as well

Screenshot:
Screenshot 2022-12-16 at 12 00 49

### CODE REVIEW

File: src/Analysis/GWASV2/Components/SelectHare/SelectHareDropDown.jsx [hunk 1]
```diff
@@ -93,17 +93,26 @@ const SelectHareDropDown = ({
...
     ).persons_in_cohort_with_value;
-    return hareBreakdownPart1 + hareBreakdownPart2;
+    return [
+      { population: 'Control', size: finalControlCohortSize },
+      { population: 'Case', size: finalCaseCohortSize },
+      { population: 'Total', size: finalControlCohortSize + finalCaseCohortSize },
+    ];
   };
 
   const onChange = (h) => {
```
Comment (jarvisraymond-uchicago) [inline src/Analysis/GWASV2/Components/SelectHare/SelectHareDropDown.jsx:118]: **Question** On line 121 we're hard coding the concept_id passed to the dispatch with action UPDATE_SELECTED_HARE. Is this correct?

File: src/Analysis/GWASV2/Steps/ConfigureGWAS/ConfigureGWAS.jsx [hunk 2]
```diff
@@ -255,10 +255,12 @@ const ConfigureGWAS = ({
               <div>Outcome Phenotype</div>
               <div>{outcome?.concept_name ?? outcome?.provided_name}</div>
             </div>
-            <div style={flexRow}>
-              <div>Final Population Size</div>
-              <div>{finalPopulationSize}</div>
-            </div>
+            {finalPopulationSizes.map((item, key) => (
+              <div key={key} style={flexRow}>
```
Comment (jarvisraymond-uchicago) [inline src/Analysis/GWASV2/Steps/ConfigureGWAS/ConfigureGWAS.jsx:259]: **Nitpick**: Using an iterator as a react key I think is discouraged (https://dev.to/devcer/why-you-should-never-use-array-index-as-key-prop-in-your-react-applications-3m9o). If we certain the items are unique, I believe you could use an item as the key.  

ESLINT isn't complaining though so its probably OK

**Review by jarvisraymond-uchicago — CHANGES_REQUESTED** [review]
Looks good, I tested locally and it works!

* I had a question about UPDATE_SELECTED_HARE hard coded value
* Had a nitpick about iterator as key that can probably be ignored since the linter doesn't seem to care. 
* Ran ESLINT-new to autoformat files, this included changes from Master. Then I updated SPRINT25 to be up-to-date with Master, but these updates are showing as changed files in this PR now; I don't think that's right; not sure why that is occurring.

File: src/Analysis/GWASV2/Components/SelectHare/SelectHareDropDown.jsx (hunk 1 above)
Comment (pieterlukasse) [reply src/Analysis/GWASV2/Components/SelectHare/SelectHareDropDown.jsx:118]: good catch. Fixed.

Comment (pieterlukasse) [reply src/Analysis/GWASV2/Components/SelectHare/SelectHareDropDown.jsx:118]: good catch.

**Review by jarvisraymond-uchicago — APPROVED** [review]
Really good!

File: src/Analysis/GWASV2/Steps/ConfigureGWAS/ConfigureGWAS.jsx (hunk 2 above)
Comment (pieterlukasse) [reply src/Analysis/GWASV2/Steps/ConfigureGWAS/ConfigureGWAS.jsx:259]: we tried to fix and failed. The error complained `[object Object]` was not unique 🤔

**Pull request merged**